doc/user_doc/vic_installation/plugin_vc_sftp.md

```diff
@@ -6,10 +6,11 @@ If your vCenter Server instance runs on a Windows system that has access to an S
 
 - You deployed at least one virtual container host to a vCenter Server instance that runs on Windows.
 - Your vCenter Server instance is running on a Windows system that has an SFTP service.
+- You must use a Windows system to run the script to install the plug-in on a vCenter Server that runs on Windows. If you used a Linux or Mac OS system to deploy the virtual container host, download and unpack the vSphere Integrated Containers Engine package on a Windows system.
```

Comment:
@hickeng Yes. I talked with Stuart about this after he tried running the `.bat` file in Linux using Wine to no avail. And the configurations will not work as expected under whatever is the choice for the Windows cmd emulator. I see that Wine is available for both Linux and macOS, but I'm not sure if we want the users to install it for this. Plus that, I need to verify the `install.bat` works in the emulator as well.